src/openvic-simulation/pop/Pop.cpp

```diff
@@ -159,9 +164,22 @@ void Pop::update_gamestate(
 	}
 }
 
-//TODO store income
-void Pop::add_rgo_owner_income(const fixed_point_t income) {}
-void Pop::add_rgo_worker_income(const fixed_point_t income) {}
+#define DEFINE_ADD_INCOME_FUNCTIONS(name)\
+	void Pop::add_##name(const fixed_point_t popIncome){\
```

Comment:
It's best practice to match parameter names in the .hpp
For the compiler it's fine, for a human reader it can be confusing.